.travis.yml

```diff
@@ -1,15 +1,20 @@
+dist: bionic
 language: go
+
 go:
-  - 1.9.x
```

Comment:
Leave the older go versions and update to include 1.13.x instead

Reply:
you got it! :)